storage/api/Storage.Samples/DownloadFile.cs

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 // [START storage_download_file]
+// [START storage_stream_file_download]
```

Comment:
@frankyn and @shaffeeullah Can you confirm that these two region tags may be covered by the same sample?

Please see this other issue and @jskeet's reply: https://github.com/googleapis/google-cloud-dotnet/issues/7638

Reply:
These region tags may be covered by the same sample 👍🏾